src/integrations/flinks.ts

```diff
@@ -0,0 +1,49 @@
+import { featheryWindow } from '../utils/browser';
+
+export async function openFlinksConnect(
+  client: any,
+  onSuccess: any,
+  flinksConfig: any,
+  updateFieldValues: any
+) {
+  const childWindow = featheryWindow().open(
+    '',
+    'Flinks Connect',
+    'width=600,height=400'
+  );
+
+  const instance = flinksConfig.metadata.instance;
+  let flinksUrl = `https://${instance}-iframe.private.fin.ag/v2/`;
+
+  if (flinksConfig.metadata.environment === 'sandbox')
+    flinksUrl += '?demo=true';
+
+  featheryWindow().addEventListener('message', async (e: any) => {
+    console.log(e.data);
+    if (e.data.step === 'REDIRECT') {
+      const loginId = new URLSearchParams(e.data.url).get('loginId');
+      client.triggerFlinksLoginId(loginId, updateFieldValues);
+      return onSuccess();
+    }
+  });
+
+  childWindow.document.write(`
+    <!DOCTYPE html>
+    <html>
+      <head>
+        <title>Child Window</title>
+      </head>
+      <body>
+        <iframe src="${flinksUrl}" width="100%" height="100%"></iframe>
```

Comment:
the iframe should expand to the full height of the child window but in your loom video it seems like it's not doing that

Reply:
@bo-dun-1 Done! Had to make the body take 100vh and 100vw!